src/components/structure/visualizing/artefact-overview/artefact-overview.tsx

```diff
@@ -1,9 +1,10 @@
-import React, { FC } from 'react';
+import React, { useContext, FC } from 'react';
```

Comment:
`useContext` kann ebenfalls wieder entfernt werden, da nirgendwo in der Komponente genutzt.